### PR TITLE
Display error popup when maximum storage limit (20MB) is reached

### PR DESCRIPTION
**Pull request by KiruthikaJeyashankar**
Card: INJI-151
Error message overlay will be displayed when the remaining storage of the device is less than or equal to 20MB

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -9,3 +9,5 @@ APPLICATION_THEME=orange
 
 #environment can be changed if it is toggled
 CREDENTIAL_REGISTRY_EDIT=true
+
+MAXIMUM_STORAGE_LIMIT_IN_MB=20
```
Comment (vijay151096) [inline .env:13]: adding MAXIMUM_STORAGE_LIMIT_IN_MB configuration in Inji.properties and using in Inji, makes application & configuration separate.

File: utils/isMaximumStorageLimitReached.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import {
+  getFreeDiskStorageOldSync,
+  getFreeDiskStorageSync,
+} from 'react-native-device-info';
+import { MAXIMUM_STORAGE_LIMIT_IN_MB } from 'react-native-dotenv';
+
+export default function isMaximumStorageLimitReached() {
+  const storageLimitInBytes = MAXIMUM_STORAGE_LIMIT_IN_MB * 1024 * 1024;
+
+  if (Platform.OS === 'android' && Platform.Version < 29) {
```
Comment (vijay151096) [inline utils/isMaximumStorageLimitReached.ts:11]: can we refactor something like this

const freeDiskStorageInBytes = (Platform.OS === 'android' && Platform.Version < 29) ? getFreeDiskStorageOldSync() : getFreeDiskStorageSync();
  return freeDiskStorageInBytes <= storageLimitInBytes;

File: .env (hunk 1 above)
Comment (KiruthikaJeyashankar) [reply .env]: Modified - fetching maximumStorageLimitInMB from configurations

File: utils/isMaximumStorageLimitReached.ts (hunk 2 above)
Comment (KiruthikaJeyashankar) [reply utils/isMaximumStorageLimitReached.ts]: Refactored

**Pull request merged**